prepare.ps1

```diff
@@ -750,16 +750,24 @@ Screensaver = 0
 New-Item $dolphinConfigFolder -ItemType directory
 Write-Output $dolphinConfigFileContent  > $dolphinConfigFile
 
+# 
+# 16. Fixing epsxe bug setting the registry
+# 
+$path = 'HKCU:\SOFTWARE\epsxe\config'
+
+New-Item -Path $path -Force | Out-Null
+
+Set-ItemProperty -Path $path -Name 'CPUOverclocking' -Value '10'
```

Comment:
Do you mind adding a URL comment here to where this is from? Will be good to refer back to later in time in-case it's fixed at some point.

Reply:
Ok. Included as comment